src/main/java/common/GamerDto.java

```diff
@@ -0,0 +1,29 @@
+package common;
+
+import domain.card.Card;
+import domain.gamer.Gamer;
+
+import java.util.List;
+
+public class GamerDto {
+    private String name;
+    private List<Card> cards;
+
+    private GamerDto(String name, List<Card> cards) {
+        this.name = name;
+        this.cards = cards;
+    }
+
+    public static GamerDto of(Gamer gamer) {
+        List<Card> cards = gamer.getPlayingCards().getCards();
```

Comment:
```suggestion
        List<Card> cards = gamer.getPlayingCards();
```

해당 클래스를 사용해는 클라이언트나 개발자에게 `PlayCards`를 공개할 필요가 없다면 바로 List 형태로 제공하는 것도 방법 중 하나입니다.
이러한 식으로 개발한다면 어떠한 차이가 있을지 고민해보아요.

Reply:
캡슐화가 더 잘되는 것 같습니다! GamerDto에서는 cards가 PlayingCards라는 일급 콜렉션으로 구현됐는 지, List로 구현됐는 지 알 필요가 없기 때문에 캡슐화가 필요한 것 같습니다. 그리고 이렇게 캡슐화를 잘 하면, PlayingCards를 일급콜렉션에서 List로 타입을 바꾸는 등의 변화가 있을 때 코드 수정을 덜해도 될 것 같습니다!